frontend/src/components/regions/AddCardReviewModal/index.jsx

```diff
@@ -33,19 +43,36 @@ function AddReviewModalUnconnected({
         }) || { loading: false }
       : { loading: false };
 
+  const handleOnChange = (e) => {
+    setIsFormValueEmpty((prev) => ({
+      ...prev,
+      comments: false,
+      status: false,
+    }));
+    const { name, value } = e.target;
+    setFormValues((prevFormValues) => ({ ...prevFormValues, [name]: value }));
+  };
+
   const handleSubmit = (e) => {
     e.preventDefault();
+
+    if (formValues.status === "")
```

Comment:
This is a strange way to do things. You are doing the validation on the frontend and then even if you know that the form is invalid, the api call gets made.

There are 2 general approaches to these sorts of things:

1. Do the validation on the frontend, and then if the frorm looks incorrect then don't make the api call. 
This is nice because it stops you from making unnecessary API calls. BUT it's also completely terrible because it means that the frontend and the backend both need to implement the same validation logic which is not at all DRY.


2. Make the api call, allow the backend to do form validation, and display any error messages that show up.
This one means that validation logic only needs to be implemented once. And if the backend's error responses are highly consistent and predictable then the frontend just needs to show the errors as they appear. We are using Django Rest Framework so the error structure is very consistent.  If there is a failed api call then we just need to display the messages that came from the server.


Let's go with 2.